### DISCUSSION TITLE
Support for non-MESH devices (lights, plugs, pebble, ...)

### DISCUSSION THREAD

**Discussion by mintgrey** [fsaris/EspHome-AwoX-BLE-mesh-hub#97] [Ideas]
Hi everybody !

I try to use this integration, but unfortunately this doesn't work with my devices, which are "non-mesh" Awox devices :(.
Since I've seen in other threads that these old devices are not supported, I don't need more help (even if any idea is welcome :), this is why I open a new discussion rather than an issue or a request.

So, the discussion is about the possibility to implement the support of this kind of devices.
**The main question :** 
If I can, I really want to help, but before digging into the code, do you know is there any unsurmountable type of difficulty, impossibility, ... to achieve this ?

To be precise, I have these devices in my own installation :
- 4 Awox lights (models SML_w7 and SML_w13), with type "ble.light.switch.dimming.white.temperature"
- 1 Awox SmartPlug, with type "ble.plug.switch"
- 1 Awox SmartPebble, with type "ble.sensor.rcu"

Thanks in advance for any response, tip or idea !

**Comment by fsaris** [comment]
You probably could try a integration like https://www.home-assistant.io/integrations/led_ble/
